src/TopSurface.cpp

```diff
@@ -122,7 +122,14 @@ bool TopSurface::ironing(const SliceMeshStorage& mesh, const GCodePathConfig& li
             }
         }
 
-        layer.addLinesByOptimizer(ironing_lines, line_config, SpaceFillType::PolyLines);
+        if(!mesh.settings.get<bool>("ironing_monotonic"))
+        {
+            layer.addLinesByOptimizer(ironing_lines, line_config, SpaceFillType::PolyLines);
+        }
+        else
+        {
+            layer.addLinesMonotonic(ironing_lines, line_config, SpaceFillType::PolyLines, (direction + 90) / 180.0 * M_PI, line_spacing * 1.1);
```

Comment:
Another raw `1.1` (and less important, another `X + 90 / 180.0 * M_PI`).

Reply:
Done: c641818be85da0f1c727ee12da58c4a7606d94c8 and ca28df1077e733d3aea67407385f017518117da4.